MekHQ/src/mekhq/IconPackage.java

```diff
@@ -163,81 +153,5 @@ public String getStartupScreenImage(int resolutionWidth) {
     public String getLoadingScreenImage(int resolutionWidth) {
         return loadingScreenImages.floorEntry(resolutionWidth).getValue();
     }
-    
-    public static Image buildForceIcon(String category, String filename, DirectoryItems items, LinkedHashMap<String, Vector<String>> iconMap) {
```

Comment:
This has been moved to `megamek.util.Images` (together with the code that processes unit icons and portraits)